sources.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "",
+  "name": "Mats Dev",
+  "categories": ["ADVERTISING"],
```

Comment:
Same thing here.

Reply:
yes those should not be there not sure why they are there since i thought i didn't add them.